src/measure_theory/set_integral.lean

```diff
@@ -464,6 +464,135 @@ lemma continuous_at.integral_sub_linear_is_o_ae
   is_o (λ s, ∫ x in s, f x ∂μ - (μ s).to_real • f a) (λ s, (μ s).to_real) ((𝓝 a).lift' powerset) :=
 (ha.mono_left inf_le_left).integral_sub_linear_is_o_ae hfm (μ.finite_at_nhds a)
 
+section continuous_linear_map
```

Comment:
section docstring, explaining what you are going to do in this paragraph, and why?